src/main/java/ScrabbleGame.java

```diff
@@ -2,26 +2,30 @@
 
 public class ScrabbleGame {
 
-    private List<String> playedWords = new ArrayList<>();
-    private List<Character> tiles = new ArrayList<>();
-    private ScrabbleDictionary dictionary = new ScrabbleDictionary();
-    private Random rnd = new Random();
+    List<String> playedWords = new ArrayList<>();
+    List<Character> tiles = new ArrayList<>();
+    private ScrabbleDictionary dictionary;
+    private LetterPool letterPool;
 
 
-    public ScrabbleGame() {
+    public ScrabbleGame(ScrabbleDictionary dictionary,
+                        LetterPool letterPool) {
+        this.dictionary = dictionary;
+        this.letterPool = letterPool;
         initializeTiles();
     }
 
     private void initializeTiles() {
         for (int i = 0; i < 7; i++) {
-            tiles.add((char) (rnd.nextInt(26) + 'a'));
+            tiles.add(letterPool.getRandomLetter());
         }
     }
 
+
     public void playGame(Scanner scanner) {
         while (true) {
             System.out.println("Enter word with given letters or 'exit' to end game\n" + tiles);
-            String word = scanner.nextLine().toLowerCase();
+            String word = scanner.nextLine().toUpperCase();
             if (word.equalsIgnoreCase("exit")) {
```

Comment:
If you convert the line to upper case, you no longer need to do equalsIgnoreCase()